.github/workflows/CI.yml

```diff
@@ -126,14 +126,13 @@ jobs:
       - name: Test dftatom
         shell: bash -e -x -l {0}
         run: |
-            git clone https://github.com/czgdp1807/dftatom.git
+            git clone https://github.com/certik/dftatom.git
             cd dftatom
-            git checkout -t origin/lf40
-            git checkout 5f54bcceab998dfc5c347eb9dfeade61c5cbc6d9
-            make -f Makefile.manual
+            git checkout b838e858e046a36dd2235713ba111199bcf718bd
+            make -f Makefile.manual F90=lfortran F90FLAGS="-I../../src"
```

Comment:
I am curious to know why we are needing to specify the imports flag? It seems we did not specify before?

Reply:
We can try it: https://github.com/lfortran/lfortran-conda-testing/pull/18, but I think it doesn't work.

Reply:
It doesn't work.

Reply:
>We can try it: https://github.com/lfortran/lfortran-conda-testing/pull/18, but I think it doesn't work.

I am curious that it used to work without import flags before, why are they being needed now?

Reply:
I think I saw it appended somehow, but I am not sure. I don't have time to investigate right now.